src/en/authors-charm-hooks.md

```diff
@@ -88,12 +88,12 @@ intervention on juju's part.
 should be used to ensure that the charm's software is not running, and will not
 start again on reboot.
 
-This hook is called when a service removal is requested by the client. It should
+This hook is called when a application removal is requested by the client. It should
```

Comment:
an application
also look for the other mulitples of times when s/service/application doesn't work :)